src/test/java/core/basesyntax/service/impl/FileWriterImplTest.java

```diff
@@ -0,0 +1,80 @@
+package core.basesyntax.service.impl;
+
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.fail;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileWriterImplTest {
+    private static final String VALID_PATH = "src\\main\\resources\\fileWriterTestFile.csv";
+    private static final String INVALID_PATH = "src/main/,resources/fileWriterTestFileOne.csv";
+    private static File resultFile;
+    private static FileWriterImpl writer;
+    private static List<String> records;
+
+    @BeforeClass
+    public static void setUp() {
+        writer = new FileWriterImpl();
+        records = List.of("fruit,quantity", "orange,56", "lemon,69", "apple,78");
+        resultFile = new File(VALID_PATH);
+    }
+
+    @Test
+    public void writeLines_shouldWriteNormally_Ok() {
+        writer.writeLines(VALID_PATH, records);
+        List<String> lines = null;
+        try {
+            lines = Files.readAllLines(resultFile.toPath());
+        } catch (IOException e) {
+            fail("Cannot read the file " + VALID_PATH);
+        }
```

Comment:
move reading logic to private method
compare results in assert, if reader helper method can't read file test will fail automatically (because you will throw runtime exception)